docs/shared/governance.md

```diff
@@ -0,0 +1,14 @@
+# Governance
+
+## Project Maintainers
+[Project maintainers](CODEOWNERS) are responsible for activities around maintaining and updating Duffle, the reference implementation for the [CNAB spec](https://github.com/deislabs/cnab-spec). Final decisions on the project reside with the project maintainers.
```

Comment:
I know this document is moved in its current form from the original repo - just wanted to point  out to update the project-specific references (_updating Duffle_, for example).

Not blocking this PR.

Reply:
I saw that, but assumed each project would just change it to their thing.

Reply:
Maybe something like:

```suggestion
[Project maintainers](CODEOWNERS) are responsible for activities around maintaining and updating <PROJECT NAME>, <PROJECT DESCRIPTION>. Final decisions on the project reside with the project maintainers.
```